Automattic/wp-calypso
# Migration: Update upgrade plan screen with CTA and title

Closes [#93297](https://github.com/Automattic/wp-calypso/issues/93297)

## Proposed Changes
* Add the `I want to import my content only` only when the upgrade step is rendered inside the new migration flow
* Use `Here’s the plan you need`  as title
* Add `MigrationUpgradePlan` to hold the flow-specific props



## Testing Instructions

* Start the new migration flow
* Go to the upgrade plan
* Click on "I want to import my content only" link
* Check if you have been redirected to the "importerWordPress" step
* Click on the back button on the step UI
* Check if you are on the `/migration-upgrade-plan`  selection page.

## Pre-merge Checklist
